atmo/jobs/tasks.py

```diff
@@ -200,7 +200,7 @@ def run_job(self, pk, first_run=False):
     spark_job = self.get_spark_job(pk)
 
     # update the cluster status of the latest Spark job run
-    updated = self.update_status(spark_job)
+    updated = self.sync_run(spark_job)
```

Comment:
Nope, that's the call to the task method `SparkJobRunTask.sync_run` which exists to be able to test this easier. In it, it calls `SparkJobRun.sync` with `commit=True`.